src/assets/avatars/Bot_With_Components.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Feels a bit odd to me that these don't live with the asset... Though it is true that keeping things up to date between branches will be problematic, you end up copying the resulting glb anyway, so this file feels like it should live with the source asset.

Reply:
> Though it is true that keeping things up to date between branches will be problematic

I still think you want to version the component data with the component. I don't think we want to tie that versioning to the assets repo.